src/index.ts

```diff
@@ -22,16 +20,15 @@ const cleanBuildDir = async (destinationDirectoryName: string): Promise<any> =>
   await emptyDir(destinationDirectoryName);
 };
 
-const compileTemplate = async (name: string, schema: OpenRPC, language: string): Promise<string> => {
-  const methodTypings = new MethodTypings(schema);
+const compileTemplate = async (openrpcDocument: OpenRPC, language: string): Promise<string> => {
+  const methodTypings = new MethodTypings(openrpcDocument);
   await methodTypings.generateTypings();
 
   const template = language === "rust" ? rsTemplate : jsTemplate;
   return template({
-    className: name,
+    className: startCase(openrpcDocument.info.title).replace(/\s/g, ""),
```

Comment:
you pulled in `trim` but didn't use it here. linting shoulda caught it

Reply:
yeah the linting for 'no unused deps' is busted.

We need to wait for eslint to work for TS. maybe it already does?